src/com/google/javascript/jscomp/SourceMapResolver.java

```diff
@@ -53,6 +60,28 @@ static SourceFile extractSourceMap(
     return getRelativePath(jsFile.getName(), sourceMapURL);
   }
 
+  /**
+   * Based on https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Data_URIs
+   * @param url
+   * @return String or null.
+   */
```

Comment:
add `@Nullable` annotation since we explicitly expect this to return null in some cases.